test-apps/display-test-app/README.md

```diff
@@ -192,12 +192,39 @@ You can use these environment variables to alter the default behavior of various
 * IMJS_ITWIN_ID.
   * GuidString of the Context Id (aka project id) to use to query Reality Data - use by Spatial Classification (e.g. "fb1696c8-c074-4c76-a539-a5546e048cc6").
   For IMJS_ITWIN_ID to work you should be in signin mode (IMJS_STANDALONE_SIGNIN=true).
+  * Also used as the iTwin ID (aka project ID) for the given iModel if IMJS_IMODEL_ID is defined.
 * IMJS_MAPBOX_KEY
   * If defined, sets the MapBox key for the `MapLayerOptions` as an "access_token".
 * IMJS_BING_MAPS_KEY
   * If defined, sets a Bing Maps key within the `MapLayerOptions` as a "key" type.
 * IMJS_CESIUM_ION_KEY
   * If defined, the API key supplying access to Cesium ION assets.
+* IMJS_IMODEL_ID
+  * If defined, the GuidString of the iModel to fetch from the iModel Hub and open.
+* IMJS_URL_PREFIX
+  * If defined, the URL prefix to use when accessing the iModel hub (eg "qa-").
+* IMJS_OIDC_HEADLESS
+  * If defined, run in headless mode. (Requires IMJS_OIDC_CLIENT_ID, IMJS_OIDC_SCOPE, IMJS_OIDC_REDIRECT_URI, IMJS_OIDC_EMAIL, and IMJS_OIDC_PASSWORD to also be defined, **or** for IMJS_OIDC_CLIENT_ID, IMJS_OIDC_CLIENT_SECRET, and IMJS_OIDC_SCOPE to be defined.)
+* IMJS_OIDC_CLIENT_ID
+  * If defined, the client ID to use for OIDC auth.
+* IMJS_OIDC_SCOPE
+  * If defined, the scope to be used for OIDC auth.
+* IMJS_OIDC_REDIRECT_URI
+  * If defined, the redirect URI to be used for OIDC auth.
+    * NOTE: as long as IMJS_OIDC_HEADLESS is not defined, OIDC auth will default to using "http://localhost:3000/signin-callback" for this.
+* IMJS_OIDC_EMAIL
+  * If defined, along with IMJS_OIDC_HEADLESS, the email address to be used for OIDC auth.
+* IMJS_OIDC_PASSWORD
```

Comment:
Can we remove all of the headless variables? I don't think they will be necessary.

Reply:
I thought one of the goals of adding hub imodel support to DTA was to allow it to run headless for automated testing.